### PR TITLE
Create Blog “converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools”

### PR DESCRIPTION
**Pull request by rondharma**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+  - api
+  - data-services-cloud-console
+---
+<!--StartFragment-->
+
+<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
+
+<!--EndFragment-->
+
+## What is HPE GreenLake APIs for data services on the HPE GreenLake Platform?
```
[the remark this answers is not in the document]
Comment (liam-m-carolan) [reply content/blog/converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools.md:19]: Perhaps this should read "...the HPE GreenLake for Data Services APIs"

**Review by liam-m-carolan — COMMENTED** [review]
Hello,

I left some suggested edits and some comments. Primarily my suggestions attempt to make some complicated sentences a little shorter and hopefully easier to read. If you want to chat through any suggestion, just let me know!

File: content/blog/converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+2. First, I deployed the Java SDK library into my Microsoft Windows desktop. My recommendation is to deploy the Microsoft Build version of the OpenJDK that is available in the Microsoft [website](https://learn.microsoft.com/en-us/java/openjdk/download) using the instruction provided from the website.  By the way, prior to deploying this OpenJDK, I removed any older version of the Java SDK or JRE from my Microsoft Windows environment so I could ensure the correct Java version used for all the steps below.
+3. Afterward, I set the `JAVA_HOME` system variable to `C:\Program Files\Eclipse Adoptium\jdk-21.0.1.12-hotspot` using the following instruction. This step ensured that any applications which require Java would use the OpenJDK by default. 
+4. Following the step above, I deployed the NodeJS package into my Microsoft Windows environment downloaded from the nodejs.org website. I also ensured that I included the **npm package manager** option as shown in one of the steps from the NodeJS installation wizard as shown in figure below.
```
[the remark this answers is not in the document]
Comment (rondharma) [reply content/blog/converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools.md]: Good catch, I made the link to the nodejs website and also fix the pointer to the instructions to install the JAVA_HOME system variable.

File: content/blog/converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+
+added 116 packages in 36s
+
+23 packages are looking for funding
+  run `npm fund` for details
+
+C:\Users\Administrator>
+```
+
+2. Afterward, I tested the deployed openapi-generator-cli to validate the version of the generator that was used. I believed that it was automatically defaulted to the latest published version which was 7.5.0 as shown below:
```
[the remark this answers is not in the document]
Comment (rondharma) [reply content/blog/converting-hpe-greenlake-api-specifications-in-oas-3-1-using-openapi-tools.md:99]: yes.. I was going to say that it's going to deploy the latest version at the time of the publication of the blog post. That version was 7.5.0

**Pull request merged**